src/index.js

```diff
@@ -2,3 +2,6 @@ export { default as v1 } from './v1.js';
 export { default as v3 } from './v3.js';
 export { default as v4 } from './v4.js';
 export { default as v5 } from './v5.js';
+export { default as REGEX } from './regex.js';
```

Comment:
We should discuss the API a bit more in depth. This would currently imply the following usage:

```js
import { REGEX } from 'uuid';
```

Having a super generic name like `REGEX` in this case is a bit unfortunate. I also find the `vX` method names already a pretty unfortunate choice because they kinda force you to do

```js
import { v4 as uuidv4 } from 'uuid';
```

in order to have a more telling method name further down in the code (who would otherwise guess that `v4()` somewhere further down in the source code generates a UUID?!).

In the [tc39/proposal-uuid](https://github.com/tc39/proposal-uuid#why-call-the-export-randomuuid-and-not-something-like-uuidv4) we therefore proposed the term `randomUUID()` instead of `v4()`.

Suggestion: `UUID_REGEXP` (also `REGEXP` instead of `REGEX` to be closer to how the JavaScript language's `RegExp`)

@awwit @broofa what do you think? (let's discuss this through before changing the code).

Reply:
FWIW, the `v4` convention came about because prior to object destructing methods were almost always referenced through the module object.  So it was typically `uuid.v4()`.  Including the module name in module properties was unnecessarily redundant.  E.g. `uuid.uuidv4()`.  But... times change.

I suggested `REGEX` because it's in keeping with the pattern we currently promote, where methods are renamed with `import ... as`.  I fully expected we'd document this with something like `import REGEX as UUID_REGEX`.

I'm not wedded to the current names (other than to suggest that we should retain backward compatibility for at least one major release).  If we can agree on a new naming scheme that works well moving forward I'm all in favor of it.  However, we'd be doing all of our users a big favor if the names ended up being consistent with whatever names the standard module proposal ends up going with. (I don't know what those are, however).

Re: REGEX vs REGEXP, the former is [far more common](https://trends.google.com/trends/explore?geo=US&q=regex,regexp).

Reply:
OK, let's not change the naming for now and stick with the currently proposed names for the first merge.